integration-cli/docker_cli_run_test.go

```diff
@@ -4567,3 +4567,14 @@ func (s *DockerSuite) TestRunServicingContainer(c *check.C) {
 	c.Assert(out2, checker.Contains, `Windows Container (Servicing)`, check.Commentf("Didn't find 'Windows Container (Servicing): %s", out2))
 	c.Assert(out2, checker.Contains, containerID+"_servicing", check.Commentf("Didn't find '%s_servicing': %s", containerID+"_servicing", out2))
 }
+
+func (s *DockerSuite) TestRunDuplicateMount(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+
+	name := "test"
+	out, _ := dockerCmd(c, "run", "--name", name, "-v", "/tmp:/tmp", "-v", "/tmp:/tmp", "busybox", "ls", "/")
+	c.Assert(out, checker.Not(checker.Contains), "tmp:")
```

Comment:
Can you add an assertion that checks whether `/tmp` is actually bind-mounted?

Maybe you should use another directory with some dummy data inside for the purpose.